src/views/AccountManager/AccountManager.js

```diff
@@ -0,0 +1,259 @@
+import React, {useContext, useState, useEffect} from "react";
```

Comment:
Started work on guest before I realized I forgot to switch branches, so sorry everything from Account Manager is here as well. Feel free to critique how badly this is designed as well if you want :P...